src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -92,7 +127,8 @@ public void testNullInfoStreamOptions() {
         final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
         try {
             final DefaultLogger logger = new DefaultLogger(outputStream, null);
-            // assert required to calm down eclipse's 'The allocated object is never used' violation
+            // assert required to calm down eclipse's 'The allocated object is never used'
+            // DefaultLogger.LocalizedMessage
```

Comment:
why you add this line ?

Reply:
I don't remember anymore why this was added. Remove now.